### PR TITLE
Global mat-raised-button changes

### PR DESCRIPTION
**Pull request by KimberleyChong**
**Description**
JIRA: [Dock-2043](https://ucsc-cgl.atlassian.net/browse/DOCK-2043?atlOrigin=eyJpIjoiNDUyODdlZjY5MzRhNDE2OThiMDE4ZTBlZTE2NjQ5NTMiLCJwIjoiaiJ9)
The purpose of this ticket was to globally update the mat-raised button style to match the redesigned buttons and remove existing styles that individually apply similar attributes.

This is going to be a long read... but in summary I'll go over:
- General Overview of Changes
- Detailed Changes and Adjustments
- Removed Classes/Styles

### General Overview of Changes
According to Zeplin there are two main sizes of buttons: large and small
Here is a  general overview of some main areas where you can see the button changes for each size:

image
image

### Detailed Changes and Adjustments
For some mat-cards, Zeplin specified a bottom padding of  8px, this was needed in several spots so a class and global style was created to add these in the necessary places:

image
image

Additional padding to launch component:

image

From my previous PR with the versions tab changes, I realised that the tools table also needed the button updates so those changes have been copied over:

image

Changes to my tools and workflows:

image

### Removed Classes/Styles
image
image
image
image

### 1.12?
So I spoke with Denis about this, if we approve these changes we can add it to the 1.12 release. Originally I planned these changes for 1.13 since they were global and I was unsure of their impact.  Now that the changes are done I don't believe anything should be a breaking change since anything with `mat-raised-button` will change to bolded text and have 1px less of right/left padding.  Additionally, it would be beneficial to be in 1.12 since my two previous PRs have buttons that are reliant on these changes (3 search buttons will now be aligned in a row and the versions tab "info" button will be up to the current mock ups)

Please make sure that you've checked the following before submitting your pull request. Thanks!

- [x] Check that your code compiles by running `npm run build`
- [x] If this is the first time you're submitting a PR or even if you just need a refresher, consider reviewing our [style guide](https://github.com/dockstore/dockstore/wiki/Dockstore-Frontend-Opinionated-Style-Guide#pr-checklist)
- [x] Do not bypass Angular sanitization (bypassSecurityTrustHtml, etc.), or justify why you need to do so
- [x] If displaying markdown, use the `markdown-wrapper` component, which does extra sanitization
- [x] Do not use cookies, although this may change in the future
- [x] Run `npm audit` and ensure you are not introducing new vulnerabilities
- [x] Do due diligence on new 3rd party libraries, checking for CVEs
- [x] Don't allow user-uploaded images to be served from the Dockstore domain

### CODE REVIEW

File: src/styles.scss [hunk 1]
```diff
@@ -1211,6 +1196,11 @@ $truncate-text-values: 1, 2, 3;
   padding-right: 1.6rem;
 }
 
+// This style should be used for the new mat-card designs that have reduced padding on bottom of card, 8px
+mat-card.mat-card-padding {
+  padding-bottom: 8px;
+}
```
Comment (garyluu) [inline src/styles.scss:1202]: have you tried not creating a new class for this and just using one of the bootstrap pb classes?

Comment (KimberleyChong) [reply src/styles.scss:1202]: mm I didn't try that, I can use pb-3 it gives me 10px if that's good with you?
image

Comment (garyluu) [reply src/styles.scss:1202]: fine with me

**Comment by KimberleyChong** [comment]
Updated mat-card padding to use bootstrap pb-3 and removed class

**Review by NatalieEO — APPROVED** [review]

File: src/assets/svg/gear-icon.svg [hunk 2]
```diff
@@ -1,3 +1,3 @@
...
+<svg width="13" height="12" viewBox="0 0 13 12" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (coverbeck) [inline src/assets/svg/gear-icon.svg:1]: What is the effect of these two lines of changes to the SVG, and why were they necessary? I'm not saying it's wrong, just looking for clarification.

Just curious, did you edit them in text, or with some sort of SVG editor.

**Review by coverbeck — APPROVED** [review]

File: src/assets/svg/gear-icon.svg (hunk 2 above)
Comment (KimberleyChong) [reply src/assets/svg/gear-icon.svg:1]: There were a few pixels of empty space around the icon which made it look slightly off centered in the buttons.
Figma! I'm more familiar with it than Zeplin, it's great for editing and exporting svgs. I also don't know of any other free svg editors

**Review by garyluu — APPROVED** [review]
assuming every button on the site has been modified to match new design

**Review by denis-yuen — APPROVED** [review]
Thanks (very thorough screenshots and style comments)!

**Comment by KimberleyChong** [comment]
yeah quick update... found some more buttons while I was working on my other button ticket.  They're all buttons contained in pop-ups that don't have Zeplin mock-ups but make sense to be bolded in order to match. Just updated their classes to match

image

**Pull request merged**